pkg/sql/alter_table.go

```diff
@@ -906,11 +905,73 @@ func applyColumnMutation(
 				col.Type.SQLString(), typ.SQLString())
 		case schemachange.ColumnConversionTrivial:
 			col.Type = *typ
-		default:
-			return unimplemented.NewWithIssueDetail(9851,
-				fmt.Sprintf("%s->%s", col.Type.SQLString(), typ.SQLString()),
-				"type conversion not yet implemented")
-		}
+
+		default: // try conversion
+			//1. Create a new computed column c' using c::t'
+			// Computed off old column, need to update ComputeExpr
+			// Make shadow column for now
+
+			// Can concurrent schema changes be happening?
+			currentMutationID := tableDesc.ClusterVersion.NextMutationID
```

Comment:
It seems fine to disable this right now, we'll just want to change this to an unimplemented with issue.